### PR TITLE
Add purging of queue

### PR DESCRIPTION
**Pull request by ingkebil**
Remove _all_ jobs from the queue.

### CODE REVIEW

**Comment by ingkebil** [comment]
Prolly should add an option to select what job type you want to purge. And then we would have to have a way to list all the job types :)

**Comment by ingkebil** [comment]
@Clinical-Genomics/reviewers  What do you think? My shortest PR so far ;)

**Comment by b4ckm4n** [comment]
Kewl, I have a n00b question though:

Is this for doing manually or something happening "under the hood"?

**Comment by ingkebil** [comment]
This is for doing it manually. It's clear a queue after you accidentally added 140 jobs like I just did ;)

**Comment by ingkebil** [comment]
Is it ok to merge?

**Comment by ingkebil** [comment]
Can't wait!

**Comment by b4ckm4n** [comment]
Merge it!

File: moneypenny/flow/cli.py [hunk 1]
```diff
@@ -36,10 +36,16 @@ def ls(context, all, limit):
...
+@click.option('-a', '--all', 'purge', is_flag=True, help='process all jobs')
 @click.pass_context
-def remove(context, job_id):
-    """Remove a job from the queue."""
+def remove(context, job_id, purge):
+    """Remove a job or all jobs from the queue."""
+    if purge:
+        db = context.obj['db']
+        while db.pop(): pass
+        return
```
Comment (robinandeer) [inline moneypenny/flow/cli.py]: should this return a non-zero exit code? 
if so use `context.abort()`

Comment (ingkebil) [inline moneypenny/flow/cli.py]: No, it should just not execute the remaining code. It has done its job and it's time to go home. Maybe there is a `context` alternative for that?

**Pull request merged**